src/content/interop/c-interop.md

```diff
@@ -260,7 +260,7 @@ use the [`package:ffigen`][ffigen] binding generator.
 ## Build and bundle native assets {:#native-assets}
 
 :::note
-The native assets are **experimental**,
+The native assets are in **preview**,
```

Comment:
Could we rephrase this as `The dart build hooks support is in preview` and also replace most other "native assets" with "dart hooks" / "dart build hooks"?